src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
+        Machine[] machines = {bulldozer, excavator, truck};
+        for (Machine machine: machines) {
+            machine.doWork();
+            machine.stopWork();
```

Comment:
Similarly, the `stopWork` method should also output an informative message indicating what type of `Machine` has stopped working. Ensure that the `stopWork` method in each specific machine class outputs a message that includes the machine type.